lib/managementClient.ts

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+import * as uuid from "uuid/v4";
+import * as debugModule from "debug";
+import { Message, ReceiverOptions, SenderOptions } from "./rhea-promise";
+import { defaultLock, translate, Constants, RequestResponseLink } from "./amqp-common";
+import { ClientEntityContext } from "./clientEntityContext";
+import { LinkEntity } from "./linkEntity";
+
+const debug = debugModule("azure:service-bus:management");
+
+export interface ManagementClientOptions {
+  address?: string;
+  audience?: string;
+}
+
+/**
+ * @class ManagementClient
+ * Descibes the EventHubs Management Client that talks
+ * to the $management endpoint over AMQP connection.
+ */
+export class ManagementClient extends LinkEntity {
```

Comment:
doesn't look like you're actually using this anywhere yet. is that accurate? assuming this is future planning for renew lock and such